src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -278,6 +279,13 @@ const formConfig = {
           uiSchema: unemployabilityFormIntro.uiSchema,
           schema: unemployabilityFormIntro.schema,
         },
+        uploadForm4192: {
+          title: 'Upload Form 4192',
+          path: 'new-disabilities/4192-upload',
+          depends: formData => formData['view:upload4192Choice'] === 'upload',
```

Comment:
Does `view:upload4192Choice` exist? I can't see anywhere where we're setting that. Did you mean `view:unemployabilityUploadChoice`?

Reply:
That will be added in a PR that is on hold because they were updating the content.